<launch>

    <!-- PREDICTION -->
    <node pkg="prediction" type="prediction.py" name="prediction" output="screen">
        <param name="dt_prediction" value="0.2" />
        <param name="time_horizon_prediction" value="4.0" />
        <param name="past_number" value="10" />
    </node>

</launch>
